participants/pServer.md

```diff
@@ -0,0 +1,74 @@
+# pServer
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name: **pServer**
+- Filecoin Slack handles:  **arsyun team**, Contact: waynewyang
+
+## (1) Brief Project Description
+
+**pServer is both a demander of ipfs/filecoin and a storage provider of ipfs/filecoin**
+
+pServer is a new product centered on storage service, with expansive function.  pServer is equivalent to a computer without screen, the APP client is equivalent to the remote desktop of the computer without screen, the user will access the pServer through APP client.
+the users of pServer can store and retrieve three types of data.
```

Comment:
Can you narrow this proposal to a single type?  I think #3 would give you the largest storage potential.  You can add the other types but the scope seems broad for this competition.